File: mocj/java.js
ocument.addEventListener("DOMContentLoaded", () => {
    const cartContainer = document.getElementById('cartContainer');
    const cartProducts = JSON.parse(localStorage.getItem('cartProducts')) || [];
  
    let totalPrice = 0;
  
    cartProducts.forEach((product, index) => {
      totalPrice += product.price;
  
      const productCard = `
        <div class="product-card">
          <img src="${product.image}" alt="${product.title}" width="100">
          <h3>${product.title}</h3>
          <p>$${product.price}</p>
          <button onclick="removeFromCart(${index})">Remove from Cart</button>
        </div>
      `;
      cartContainer.innerHTML += productCard;
    });
  
    document.getElementById('totalPrice').querySelector('h3').textContent = Total Price: $${totalPrice};
  });
  
  function removeFromCart(index) {
    let cartProducts = JSON.parse(localStorage.getItem('cartProducts')) || [];
    cartProducts.splice(index, 1);
    localStorage.setItem('cartProducts', JSON.stringify(cartProducts));
    window.location.reload();
  }
  
  document.getElementById('checkoutBtn').addEventListener('click', () => {
    if (confirm("Are you sure to checkout?")) {
      setTimeout(() => {
        alert("Thanks for shopping!");
        localStorage.removeItem('cartProducts');
        window.location.reload();
      }, 2000);
    }
  });

File: mocj/java.java
<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <title>Cart</title>
  <link rel="stylesheet" href="css/styles.css">
</head>
<body>
  <!-- Navbar -->
  <nav>
    <a href="index.html">Home</a>
    <a href="wishlist.html">Wishlist</a>
    <a href="cart.html">Cart</a>
  </nav>

  <h1>Your Cart</h1>

  <div id="cartContainer" class="product-container"></div>

  <div id="totalPrice">
    <h3>Total Price: $0</h3>
    <button id="checkoutBtn">Checkout</button>
  </div>

  <script src="js/cart.js"></script>
</body>
</html>